node/src/reactor.rs

```diff
@@ -497,7 +497,13 @@ where
             );
         }
 
-        let scheduler = utils::leak(Scheduler::new(QueueKind::weights()));
+        let event_queue_dump_threshold =
+            env::var("EVENT_QUEUE_DUMP_THRESHOLD").map_or(None, |s| s.parse::<usize>().ok());
```

Comment:
This env var should be documented in the top-level readme, under "Other environment variables" I guess.  I'd been trying to use a convention of `CL_` prefix for Casper env vars, but I don't think that has stuck, so don't worry about renaming this one :)

Reply:
Variable is renamed and readme is updated in https://github.com/casper-network/casper-node/pull/4295/commits/e44daa4b3fa3d3c0b0993f78368fd248c276345c